src/main/webapp/app/detail-overview-list/detail-overview-list.component.html

```diff
@@ -102,9 +102,7 @@ <h3 class="section-headline" [id]="headlinesRecord[section.headline]">{{ section
                                                 </a>
                                             }
                                             <div class="auxiliaryRepositoryDescription">
-                                                @if (auxiliaryRepository.checkoutDirectory) {
```

Comment:
We do not need to display the checkout directory twice _(you can now retrieve this information from the checkout directory table)_